tgbotapi.api/src/commonMain/kotlin/dev/inmo/tgbotapi/extensions/api/edit/Edits.kt

```diff
@@ -99,240 +99,240 @@ suspend fun TelegramBot.edit(
     proximityAlertRadius: Meters? = null,
     businessConnectionId: BusinessConnectionId? = message.chat.id.businessConnectionId,
     replyMarkup: InlineKeyboardMarkup? = null
-) = editLiveLocation(message, latitude, longitude, livePeriod, horizontalAccuracy, heading, proximityAlertRadius, businessConnectionId, replyMarkup)
+): ContentMessage<LocationContent> = editLiveLocation(message, latitude, longitude, livePeriod, horizontalAccuracy, heading, proximityAlertRadius, businessConnectionId, replyMarkup)
 
 /**
  * @param replyMarkup Some [InlineKeyboardMarkup]. See [dev.inmo.tgbotapi.extensions.utils.types.buttons.inlineKeyboard]
  * as a builder for that
  */
-suspend fun TelegramBot.edit(
+public suspend fun TelegramBot.edit(
     chatId: ChatIdentifier,
     messageId: MessageId,
     location: LiveLocation,
     businessConnectionId: BusinessConnectionId? = chatId.businessConnectionId,
     replyMarkup: InlineKeyboardMarkup? = null
-) = editLiveLocation(
+): ContentMessage<LocationContent> = editLiveLocation(
```

Comment:
Are you sure that here `ContentMessage<LocationContent>` and not `ContentMessage<LiveLocationContent>`?

Reply:
see other response